src/app/access-control/group-registry/groups-registry.component.html

```diff
@@ -47,7 +47,7 @@ <h3 id="search" class="border-bottom pb-2">{{messagePrefix + 'search.head' | tra
             <thead>
             <tr>
               <th scope="col">{{messagePrefix + 'table.id' | translate}}</th>
-              <th scope="col">{{messagePrefix + 'table.name' | translate}}</th>
+              <th scope="col" colspan="2">{{messagePrefix + 'table.name' | translate}}</th>
```

Comment:
Instead of making this a `colspan="2"`, we should add a separate `<th>` that says "Collection/Community".  The current display is confusing, as it appears the collection is part of the group "Name" column.